src/kdc/do_as_req.c

```diff
@@ -240,10 +240,8 @@ finish_process_as_req(struct as_req_state *state, krb5_error_code errcode)
     state->reply.ticket = &state->ticket_reply;
     state->reply_encpart.session = &state->session_key;
     if ((errcode = fetch_last_req_info(state->client,
-                                       &state->reply_encpart.last_req))) {
-        state->status = "FETCH_LAST_REQ";
+                                       &state->reply_encpart.last_req)))
```

Comment:
That would be out of scope for this PR.  Also, I hope that we'll flesh out fetch_last_req_info() some day, but as it already has a DB entry to work with, it should be an only-fail-on-ENOMEM type operation, so we shouldn't need a status code for it.